docs/console/company-configuration/providers/extension-providers.md

```diff
@@ -0,0 +1,40 @@
+---
+id: extension-providers
+title: Extension Providers
+sidebar_label: Extension Providers
+---
+
+:::info
+
+Extension Providers can only be used when the [Enhanced Project Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) is enabled.
+
+:::
+
+Some providers can extend or replace specific Console features.
+
+The steps to enable a Project Extension, using an Extension Provider, are the following:
+
+- check that the [Enhanced Project Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) is enabled for the project; 
```

Comment:
this is not required to create an Extension API Provider

Reply:
moved to Orchestrator Generator page